cost/azure/rightsize_sql_instances/azure_rightsize_sql_instances.pt

```diff
@@ -333,10 +324,13 @@ end
 
 policy 'policy_azure_db_utilization' do
   validate $ds_merged_metrics do
-    summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} Azure Rightsize SQL Databases\n
-    ### Thresholds for Consideration
-    - Upsize Average CPU% threshold   : {{ parameters.param_avg_cpu_upsize }}
-    - Downsize Average CPU% threshold : {{ parameters.param_avg_cpu_downsize }}"
+    summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): Found {{ len data }} Azure Rightsize SQL single Databases"
+	detail_template <<-EOS
+      ### Thresholds for Consideration
+      - Upsize Average CPU% threshold   : {{ parameters.param_avg_cpu_upsize }}
+      - Downsize Average CPU% threshold : {{ parameters.param_avg_cpu_downsize }}
+	  Note: If Recommendation column contains the value "Change tier", then it is recommended to change service tier manually since different service tiers have different capabilities and costs
```

Comment:
This can probably be removed since its in the readme.

Reply:
Updated